public/app/features/dashboard/components/Inspector/DetailText.tsx

```diff
@@ -0,0 +1,16 @@
+import React, { FC } from 'react';
+import { useStyles } from '@grafana/ui';
+import { GrafanaTheme } from '@grafana/data';
+import { css } from 'emotion';
+
+const getStyles = (theme: GrafanaTheme) => css`
+  margin: 0;
+  margin-left: ${theme.spacing.md};
+  font-size: ${theme.typography.size.sm};
+  color: ${theme.colors.textWeak};
+`;
+
+export const DetailText: FC = ({ children }) => {
+  const collapsedTextStyles = useStyles(getStyles);
```

Comment:
@tskarhed can you bring the `useStyles` to upcoming front end weekly? :)

Reply:
Oh, right! Yeah, I created it a couple of months ago because people constantly used the same pattern.